Tests/StencilSwiftKitTests/StringFiltersTests.swift

```diff
@@ -170,7 +170,7 @@ extension StringFiltersTests {
       XCTFail("Code did succeed while it was expected to fail for wrong option")
     } catch Filters.Error.invalidOption {
       // That's the expected exception we want to happen
-    } catch let error {
+    } catch {
```

Comment:
It gives a warning if the `catch let error` doesn't add anything. If you want explicit, we should use `catch let error as Error`.

Reply:
Yeah I got that and that's not the style I want / am used to.

I'm used to `catch let error` without type when we catch all errors because that way you understand where that `error` variable is coming from, while with just `catch` it's just magically defined and appears from nowhere explicitly.
But I feel like `catch let error as Error` isn't adding much and might even be confusing, feels a bit like `if let x = y as Any` vs `if let x = y`

So overall I personally don't like the rule altogether. I would even wish there were an opposite rule forcing the presence of explicit `catch let error` as soon as `error` is used in the catch block.